libs/sdk-backend-spi/src/organization/settings/index.ts

```diff
@@ -26,6 +26,24 @@ export interface IOrganizationSettingsService {
      */
     setLocale(locale: string): Promise<void>;
 
+    /**
+     * Sets timezone for organization.
+     *
+     * @param locale - for example "en-US", "cs-CZ", etc.
```

Comment:
This description seems to be invalid, doesn't it?

→ Please update this accordingly.

Reply:
fixed